tutorial/admin/README.md

```diff
@@ -12,7 +12,71 @@ The directory contains scripts that can be used to configure systems
 for use with the GEOPM runtime.  The scripts associated with each
 section are numbered accordingly.
 
-0. Enable users to do research with the GEOPM runtime
+0. Set up a system to use the GEOPM static policy plugin for SLURM
+-------------------------------------------------------------------
+GEOPM provides two libraries: libgeopmpolicy contains tools for interacting
+with hardware signals and controls, such as geopmread, and the supporting
+library functions.  libgeopm contains all the same functions and additionally
+provides tools for launching and interacting with MPI applications. The slurm
+plugin requires libgeopmpolicy only; it does not use libgeopm or MPI.
+If the GEOPM runtime will be installed (libgeopm and MPI launch tools), it
+should be installed in a different location, preferably using the module system.
+The packages are available through OpenHPC (https://openhpc.community/downloads/).
+See step 1 in this guide.
+
+A.  To build libgeopmpolicy and a compatible libgeopm_slurm.so:
+    - Obtain the source code for geopm from https://github.com/geopm/geopm
+    - Make sure no modules are loaded in the environment that will affect
+      compilation:
+        ```
+        module purge
+        ```
+    - Run configure with a local prefix that will be used to build the plugin:
+        ```
+        ./configure --disable-mpi --disable-fortran --prefix=$HOME/build/geopm-no-mpi
+        ```
+    - Build and install for use when building the slurm plugin
+        ```
+        make -j && make install
+        ```
+    - Build an RPM to be used to install geopmpolicy on the compute nodes:
+        ```
+        make rpm
+        ```
+    - Obtain the source code for the plugin from https://github.com/geopm/geopm-slurm
+    - Configure using --with-geopm targeting the geopm install above:
+        ```
+        ./autogen.sh
+        ./configure --with-geopm=$HOME/build/geopm-no-mpi --prefix=$HOME/build/geopm-slurm-plugin
+        ```
+    - Build the plugin:
+        ```
+        make && make install
+        ```
+B.  Install the libgeopmpolicy package in the compute nodes using the RPM.
+The library must be accessible to the root user when the plugin runs,
+for example in /usr/lib64, and this version of the library should be built
+against a toolchain available in the system default paths.
+
+C.  Install libspank_geopm.so* into /usr/lib64/slurm on the compute nodes.
+
+D.  Create or update plugstack.conf in /etc/slurm on the compute nodes
+only with the following:
+    ```
+    optional  libgeopm_spank.so
+    ```
+The head node should not need plugstack.conf; or if present, it should not
+contain any reference to libgeopm_spank.so.
+
+E.  Update the Slurm configuration (/etc/slurm/slurm.conf) as follows:
+    - Update PluginDir to contain "/usr/lib64/slurm"
+    - Make sure that PlugStackConfig is not set.
```

Comment:
This is OK imo.  If the admin wants to put the plugstack.conf somewhere other than the default, that's OK as long as libgeopm_spank.so is present in the file.